specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-11-01/webapplicationfirewall.json

```diff
@@ -179,6 +179,56 @@
         },
         "x-ms-long-running-operation": true
       },
+      "patch": {
```

Comment:
My change is adding path api, might not be breaking changes. @lirenhe Renhe Li How do you think? Thanks!

Reply:
Based on our policy https://github.com/Azure/openapi-diff/tree/master/docs/rules/1039.md, this is still a breaking change and you need to follow the process to get it reviewed.

Reply:
@lirenhe why is adding a new API a breaking change? The link you provided does not explain why this policy exists. How is adding a new API breaking any clients?

Reply:
It is based on http://aka.ms/bcforapi and if you have question, please go to office hours listed in [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).